mastermelon/melon.py

```diff
@@ -455,6 +455,24 @@ async def convertexp(ctx: discord.ext.commands.Context):
         else:
             await ctx.channel.send("You have no exp. ;-; Can't convert emptiness.")
 
+@bot.command
+async def appeal(ctx, args, punishment: str, idoruuid: str = None, reason: str = None):
+     if isinstance(idoruuid, type(None)):
+        return ctx.channel.send("you must fill a player id or uuid")
+     if not punishment.startwith("minecraftBan", "terrariaBan", "mindustryKick", "mindustryBan"):
+        return ctx.channel.send("you must fill a punishment type, here is:/nmindustryBan, mindustryKick, terrariaBan, minecraftBan")
+     if isinstance(reason, type(None)):
+        return ctx.channel.send("you must fill a reason of you got banned/kick")
+     else:
+          channel = discord.utils.get(ctx.guild.channels, id="")
```

Comment:
Add a channel via channelid

Reply:
on line 467

Reply:
![image](https://user-images.githubusercontent.com/67626131/107845674-50df1600-6e18-11eb-917c-68d406dd0c9b.png)

Reply:
![image](https://user-images.githubusercontent.com/67626131/107845803-46714c00-6e19-11eb-829e-6ef5a5ea4cac.png)
2) players dont know their uuid
1) missing id, will break the code.

Reply:
ok